components/script/dom/htmlscriptelement.rs

```diff
@@ -212,8 +220,33 @@ impl PreInvoke for ScriptContext {}
 /// https://html.spec.whatwg.org/multipage/#fetch-a-classic-script
 fn fetch_a_classic_script(script: &HTMLScriptElement,
                           url: Url,
+                          cors_setting: Option<CORSSettings>,
                           character_encoding: EncodingRef) {
-    // TODO(#9186): use the fetch infrastructure.
+    let doc = document_from_node(script);
+
+    // Step 1, 2.
+    let request = RequestInit {
+        method: Method::Get,
```

Comment:
Why are we initializing most of these fields?

> Let request be the result of creating a potential-CORS request given url, "script", and CORS setting.

> To create a potential-CORS request, given a url, destination, corsAttributeState, and an optional same-origin fallback flag

Only those four fields need to be set. If you want, add a method `RequestInit::potential_cors(url, dest, corsAttributeState,Option<FalbackFlag>)` or something and use it here.